domino_generator/example/ex1/ex1.g.dart

```diff
@@ -2,14 +2,16 @@ import 'package:domino/src/experimental/idom.dart' as _i0 show DomContext;
 import 'package:meta/meta.dart' as _i2 show required;
 import 'ex1_model.dart' as _i1 show Example;
 
-void renderEx1(
+void Ex1(
   _i0.DomContext $d, {
 
   /// Go recursive
   bool extra,
   @_i2.required _i1.Example obj,
+  Map<String, void Function(_i0.DomContext)> $dSlots,
```

Comment:
It would be nice to have the named slots arguments,  similarly to the other ones (e.g. `extra`, `obj`).

It can be of the following type:
````
typedef SlotFn = void Function(DomContext d);
````